Firebase/Auth/Source/FIRAuthAPNSTokenManager.m

```diff
@@ -73,7 +73,9 @@ - (void)getTokenWithCallback:(FIRAuthAPNSTokenCallback)callback {
     } else {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#if !TARGET_OS_TV
```

Comment:
Fixed! The check is necessary since UIApplication doesn't have a `registerForRemoteNotificationTypes:` method available in tvOS so this fails to compile unless it's wrapped.

Reply:
Gotcha :) thanks for the background.